index.js

```diff
@@ -135,21 +136,43 @@ const goodserverPost = async (cmd, data, env) => {
   return response.json()
 }
 
-const travisPost = async (repocmd, data) => {
-  const tosend = {
-    request: data,
+const githubPost = async (
+  releaseType,
+  sourceBranch,
+  targetBranch,
+  repo,
+  workflowId,
+) => {
+  console.log('slack release github action:', {
+    releaseType,
+    sourceBranch,
+    targetBranch,
+    repo,
+    workflowId,
+  })
+
+  const authToken = btoa(GITHUB_USERNAME + ':' + GITHUB_TOKEN)
+  const body = {
+    ref: sourceBranch,
+    inputs: {
+      release: releaseType,
+      targetbranch: targetBranch,
+    },
   }
-  const response = await fetch('https://api.travis-ci.com/repo/' + repocmd, {
-    method: 'POST', // *GET, POST, PUT, DELETE, etc.
-    headers: {
-      'Content-Type': 'application/json',
-      'Travis-API-Version': 3,
-      Authorization: `token ${TRAVIS_TOKEN}`,
+  const res = await fetch(
```

Comment:
a raw request again, maybe it's time to move to the axios ?
- create & configure root API objects
- then just use api.<method>('/relative/endpoint', { data as JS object, will be automatically strginified })

If axios won't work in the worker (i remember you've told about som issues with it) maybe to find some lightweight alternative which allows to create pre-configured api objects

Reply:
I am not sure if anything else works in the worker environment
maybe it is possible to wrap it with some kind of method to make it simpler but I think it is over-engineering for this project